wdl/GatherBatchEvidenceMetrics.wdl

```diff
@@ -30,6 +30,19 @@ workflow GatherBatchEvidenceMetrics {
     String sv_pipeline_base_docker
     String linux_docker
 
+    RuntimeAttr? runtime_attr_delly_metrics
+    RuntimeAttr? runtime_attr_manta_metrics
+    RuntimeAttr? runtime_attr_melt_metrics
+    RuntimeAttr? runtime_attr_wham_metrics
+    RuntimeAttr? runtime_attr_baf_metrics = {"mem_gb": 30, "disk_gb": 100}
+    RuntimeAttr? runtime_attr_sr_metrics = {"mem_gb": 30, "disk_gb": 100, "preemptible_tries": 0}
+    RuntimeAttr? runtime_attr_pe_metrics = {"mem_gb": 15, "disk_gb": 100, "preemptible_tries": 0}
```

Comment:
This might cause problems with our mini-wdl friends. Let's just modify the task defaults instead.